components/translations/translations.en.js

```diff
@@ -105,6 +114,16 @@ export const translations = {
   'cc-elasticsearch-info.link.elasticsearch': `Check out the Elasticsearch add-on`,
   'cc-elasticsearch-info.link.kibana': `Open Kibana`,
   'cc-elasticsearch-info.text': `This service is part of the Elasticsearch Enterprise offer. You can find the documentation and linked services right below.`,
+  // cc-elasticsearch-options
+  'cc-elasticsearch-options.enabled': `Enabled`,
+  'cc-elasticsearch-options.disabled': `Disabled`,
+  'cc-elasticsearch-options.confirm': `Confirm options`,
+  'cc-elasticsearch-options.title': `Options for Elasticsearch Enterprise`,
+  'cc-elasticsearch-options.description': () => sanitize`This add-on is part of our Elasticsearch Enterprise offer which include two options. Both these options will be deployed as application and managed and updated by Clever Cloud on your behalf. They will appear as regular applications that you can stop, scale up or down automatically like a regular app. As such <strong>enabling these options will result in an increase in credits consumption</strong> as well.`,
+  'cc-elasticsearch-options.description.kibana': () => sanitize`Kibana is the admin UI for the Elastic Stack. It lets you visualize your Elasticsearch data and navigate the Stack so you can do anything from tracking query load to understanding the way requests flow through your apps. Learn more on the <a href="https://www.elastic.co/guide/en/kibana/master/index.html">official Kibana documentation</a>.`,
+  'cc-elasticsearch-options.description.apm': () => sanitize`Elastic APM server is an application performance monitoring system built on the Elastic Stack. Deploying this will allow you to automatically send APM metrics from any applications linked to this Elastic addon instance, providing you add the Elastic APM agent to the application code. Learn more on the <a href="https://www.elastic.co/guide/en/apm/get-started/current/overview.html">official APM server documentation</a>.`,
```

Comment:
```suggestion
  'cc-elasticsearch-options.description.apm': () => sanitize`Elastic APM server is an application performance monitoring system built on the Elastic Stack. Deploying this will allow you to automatically send APM metrics from any applications linked to this Elastic addon instance, providing you add the Elastic APM agent to the application code. Learn more in the <a href="https://www.elastic.co/guide/en/apm/get-started/current/overview.html">official APM server documentation</a>.`,
```